portal-web/docroot/html/portlet/layouts_admin/edit_layout_set.jsp

```diff
@@ -69,43 +61,9 @@ boolean hasViewPagesPermission = (pagesCount > 0) && (liveGroup.isStaged() || se
 
 			<aui:nav-item href="<%= addPagesURL %>" iconCssClass="icon-plus" label="add-page" />
 		</c:if>
-		<c:if test="<%= hasExportImportLayoutsPermission %>">
-			<liferay-portlet:renderURL portletName="<%= PortletKeys.EXPORT_IMPORT %>" var="exportPagesURL" windowState="<%= LiferayWindowState.POP_UP.toString() %>">
-				<portlet:param name="struts_action" value="/export_import/export_layouts" />
-				<portlet:param name="<%= Constants.CMD %>" value="<%= Constants.EXPORT %>" />
-				<portlet:param name="groupId" value="<%= String.valueOf(groupId) %>" />
-				<portlet:param name="liveGroupId" value="<%= String.valueOf(liveGroupId) %>" />
-				<portlet:param name="privateLayout" value="<%= String.valueOf(privateLayout) %>" />
-				<portlet:param name="rootNodeName" value="<%= rootNodeName %>" />
-				<portlet:param name="showHeader" value="<%= Boolean.FALSE.toString() %>" />
-			</liferay-portlet:renderURL>
-
-			<aui:nav-item href="<%= exportPagesURL %>" iconCssClass="icon-arrow-down" label="export" useDialog="<%= true %>" />
-
-			<liferay-portlet:renderURL portletName="<%= PortletKeys.EXPORT_IMPORT %>" var="importPagesURL" windowState="<%= LiferayWindowState.POP_UP.toString() %>">
-				<portlet:param name="struts_action" value="/export_import/import_layouts" />
-				<portlet:param name="<%= Constants.CMD %>" value="<%= Constants.VALIDATE %>" />
-				<portlet:param name="groupId" value="<%= String.valueOf(groupId) %>" />
-				<portlet:param name="privateLayout" value="<%= String.valueOf(privateLayout) %>" />
-				<portlet:param name="rootNodeName" value="<%= rootNodeName %>" />
-				<portlet:param name="showHeader" value="<%= Boolean.FALSE.toString() %>" />
-			</liferay-portlet:renderURL>
-
-			<aui:nav-item href="<%= importPagesURL %>" iconCssClass="icon-arrow-up" label="import" useDialog="<%= true %>" />
-		</c:if>
 	</aui:nav>
 </aui:nav-bar>
 
-<c:if test="<%= liveGroup.isStaged() %>">
```

Comment:
I think removing this part from the code removes the warning message that this is a staged environment, this might cause confusion to the end users if they don't see it. I can see this is removing coupling to reach the files in the export_import portlet, but I think it might be good to retain the warning message